barretenberg/acir_tests/bootstrap.sh

```diff
@@ -124,11 +124,6 @@ function test_cmds_internal {
   echo SYS=ultra_honk FLOW=prove_then_verify RECURSIVE=true $run_test double_verify_honk_proof
   echo SYS=ultra_honk FLOW=prove_then_verify HASH=keccak $run_test assert_statement
   echo SYS=ultra_honk FLOW=prove_then_verify ROLLUP=true $run_test verify_rollup_honk_proof
-
```

Comment:
These tests are going to be removed by [this](https://github.com/AztecProtocol/aztec-packages/pull/12126) PR from Cody/Adam. They represent test-only functionality that we never had plans to support.